code/obj/critter/critter_parent.dm

```diff
@@ -14,6 +14,7 @@
 	var/is_template = 0
 	var/alive = 1
 	var/health = 10
+	var/max_health = 100
```

Comment:
Could you merge the drone maxhealth var with this var?
Also if you are adding this could you make it so healing in this file also respects max_health? (There's two instances of food healing here.)